docs/how-to-guides/running-feast-in-production.md

```diff
@@ -144,3 +185,142 @@ prediction = model.predict(feature_vector)
 It is important to note that both the training pipeline and model serving service only needs read access to the feature registry and associated infrastructure. This prevents clients from accidentally making changes to the feature store.
 {% endhint %}
 
+## 4. Retrieving online features for prediction
+
+Once you successfully loaded (or in Feast terminology materialized) your data from Batch Source into Online Store you can start consuming features for inference phase.
+There are three approaches for that purpose sorted from the most simple one (in operational sense) to the most performant:
+
+### 4.1. Python SDK reads directly from Online Store
```

Comment:
nit: maybe 4.1 Use the python SDK within an existing python service?

also, the language in "But since a lot of Data Science tools are already using Python, and there's a great chance your service will be using it as well, this approach is still pretty valid for production."  is a bit wordy. "still pretty valid for production" also seems mysterious haha. It's also not clear whether you mean that there are a lot of python DS tools that can be useful in production or if the infrastructure engineer might be more comfortable in python because of prior data science exposure.

maybe something like "A benefit to using a python stack is that you can enjoy production grade services with integrations with many existing data science tools"